.gitignore

```diff
@@ -0,0 +1 @@
+/test.md
```

Comment:
- Kindly add `node_modules` to the `.gitignore` file as it's in the requirement of the project.

Reply:
Hi George. In my requirements list, I can't see the instruction that says that I must add node_modules to the gitignore file. Can you please help me find it? Kindly see all the requirements below:

Requirements
Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
Create a new GitHub repo.
Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
Add a .gitignore file.
Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
Add 3 files (each one in a separate commit):
A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
A .css file that includes one class with styles for your header (make it in your favorite color).
A test.md file with the text "This file should be ignored by git".
Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
Open a pull request.

Reply:
Hello @rolandfbk,
This is mentioned in linters repo, you can view it [here](https://github.com/microverseinc/linters-config/tree/master/html-css#:~:text=Note%3A%20The%20npm%20package%20manager%20is%20going%20to%20create%20a%20node_modules%20directory%20to%20install%20all%20of%20your%20dependencies.%20You%20shouldn%27t%20commit%20that%20directory.%20To%20avoid%20that%2C%20you%20can%20create%20a%20.gitignore%20file%20and%20add%20node_modules%20to%20it%3A)